src/pages/components/chatbot/widgets/Requirements.js

```diff
@@ -3,8 +3,8 @@ import { requirements } from '../../../../utilities/constants';
 
 const Requirements = () => {
   return (
-    <div className="border-2 p-4 pl-9 mx-12 bg-sky-100 rounded-lg">
-      <ul className=" list-disc">
+    <div className='border-2 p-4 pl-9 mx-12 bg-sky-100 rounded-lg'>
+      <ul className='list-disc'>
         {requirements.map((req) => {
           return (
             <li className="pb-3" key={req.id}>
```

Comment:
```suggestion
            <li className='pb-3' key={req.id}>
```
change to single quote